projects/table-builder/src/lib/interfaces/report-def.ts

```diff
@@ -75,7 +76,8 @@ export interface Additional   {
     header?: Dictionary<string>,
     body?: Dictionary<string>,
     footer?: Dictionary<string>,
-  } 
+  }
+  enumMap?: any;
```

Comment:
maybe Dictionary<string | number>

Reply:
I'm not sure what you're gaining? an enum should always match a number to a string. (Even if typescript doesn't work that way but for our purposes that's how we're using it)

Reply:
the way that we are using it, it should typed as Dictionary<string>